### ISSUE TITLE
test http_client.test.lua fails on Mac

### ISSUE THREAD

**Issue by OKriw** [tarantool/tarantool#3564]
While running tests on mac os - I got one failed test: http_client
```
./test/app-tap/http_client.test.lua
TAP version 13
# TARANTOOL_SRC_DIR=../..
1..2
    # http over AF_INET
    1..9
    # starting HTTP server on 127.0.0.1:49678...
sh: ../../test/app-tap/httpd.py: No such file or directory
    not ok - server started
      ---
```
If TARANTOOL_SRC_DIR is not set "../.." is used for path to look for httpd.py
if I change line to: local TARANTOOL_SRC_DIR = os.getenv("TARANTOOL_SRC_DIR") or "." 
````
# TARANTOOL_SRC_DIR=.
1..2
    # http over AF_INET
    1..9
    # starting HTTP server on 127.0.0.1:49749...
./test/app-tap/httpd.py --inet 127.0.0.1:49749
    ok - server started
    # trying to connect to http://127.0.0.1:49749/
    ok - connection is ok
        # http.client
        1..9
        ok - global namespace is not polluted
        ok - global namespace is not polluted
        ok - simple 200
        ok - proto major http 1.1
        ok - proto major http 1.1
        ok - body
        ok - content-length > 0
Bus error: 10
```
So, we should find another way to get path for httpd.py.

**Comment by Totktonada** [comment]
Seems to be dupe of #3418, #3973 and #4005. The following patch helps me on a Mac OS machine:

```diff
diff --git a/src/lib/core/fiber.c b/src/lib/core/fiber.c
index abd6c6b11..529acf2d5 100644
--- a/src/lib/core/fiber.c
+++ b/src/lib/core/fiber.c
@@ -95,7 +95,7 @@ enum {
        /* The minimum allowable fiber stack size in bytes */
        FIBER_STACK_SIZE_MINIMAL = 16384,
        /* Default fiber stack size in bytes */
-       FIBER_STACK_SIZE_DEFAULT = 65536
+       FIBER_STACK_SIZE_DEFAULT = 8388608,
 };
 
 /** Default fiber attributes */
```

**Comment by Totktonada** [comment]
See also #4179. Now the test passes. I'll drop `sergw/fix-osx-http-client-test` branch ( bcab4b134cb94b068379d03c785aa45cf5a26671 ).
